build-cluster-ec2.yml

```diff
@@ -93,10 +93,14 @@
   gather_facts: True
   sudo: True
 
-# Get these from env to workaround https://github.com/ansible/ansible/issues/10638
   vars:
+# Get these from env to workaround https://github.com/ansible/ansible/issues/10638
     aws_access_key:  "{{ lookup('env','AWS_ACCESS_KEY_ID') }}"
     aws_secret_key:  "{{ lookup('env','AWS_SECRET_ACCESS_KEY') }}"
+# These are the default ConductR roles - these roles may be overridden within the VARS_FILE
+    CONDUCTR_ROLES:
+      - web
+      - haproxy
```

Comment:
@edwardcallahan - these are the default ConductR roles which may be overridden via VARS_FILE as such:

```
CONDUCTR_ROLES:
  - web
  - haproxy
  - elasticsearch
```